lib/pa_ess/utilities.ex

```diff
@@ -520,6 +524,13 @@ defmodule PaEss.Utilities do
 
   def prediction_four_cars?(_), do: false
 
+  def prediction_alewife_braintree?(%Predictions.Prediction{
+        stop_id: stop_id
+      }),
+      do: stop_id in ["70105", "70061"]
```

Comment:
This technically works, but not in the way that's intended. Currently the second function clause below will never get called, because this one will always match, and will just return the result of the `in` check. If we want to express this as a multi-clause function (using the fallback below), then we'd need to write the `in` as a guard instead (using `when`)